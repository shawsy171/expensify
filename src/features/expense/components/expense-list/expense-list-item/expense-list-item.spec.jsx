import React from 'react';
import { shallow } from 'enzyme';
import moment from 'moment';

import ExpenseListItem from './expense-list-item';

describe('ExpenseListItem Component', () => {
  test('should render an item', () => {
    const expense = {
      id: '3',
      description: 'Credit Card',
      note: '',
      amount: 4499,
      createdAt: moment(0).add(4, 'days').valueOf(),
    };

    const wrapper = shallow(<ExpenseListItem expense={expense} />);
    expect(wrapper).toMatchSnapshot();
  });
});
