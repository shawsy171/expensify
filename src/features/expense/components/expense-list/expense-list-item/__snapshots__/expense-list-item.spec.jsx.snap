// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExpenseListItem Component should render an item 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/3"
  >
    <h3>
      Credit Card
    </h3>
  </Link>
  <p>
    4500
     - 
    345600000
  </p>
</div>
`;
